projects/fsastorefrontlib/src/cms-components/my-account/premium-calendar/premium-calendar.component.html

```diff
@@ -1,238 +1,134 @@
 <h2 class="heading-headline py-3 mt-3 mb-5 text-center">
   {{ 'premiumCalendar.premiumCalendar' | cxTranslate }}
 </h2>
-<ng-container *ngIf="policiesLoaded$ | async; else loading">
-  <ng-container *ngIf="displayPolicies?.length > 0; else noPolicies">
-    <div class="container-fluid d-none d-lg-block">
-      <!-- Desktop Layout-->
-      <div class="section-header-heading row">
-        <h5 class="col">{{ 'policy.policy' | cxTranslate }}</h5>
-        <h5 class="col">{{ 'fscommon.paymentFrequency' | cxTranslate }}</h5>
-        <h5 class="col">{{ 'premiumCalendar.dueDate' | cxTranslate }}</h5>
-        <h5 class="col">{{ 'fscommon.premium' | cxTranslate }}</h5>
-        <h5 class="col">{{ 'fscommon.paymentMethod' | cxTranslate }}</h5>
-      </div>
-      <div *ngFor="let policy of displayPolicies">
-        <div
-          class="premium-data-row row border boder-color-3"
-          (click)="openPolicy(policy)"
-          [ngClass]="{
-            clickable: policy.billingData?.length > 1,
-            active: policy.opened
-          }"
-        >
-          <div class="col">
-            <div class="row">
-              <div class="col-4">
-                <div class="image-wrapper">
-                  <img
-                    class="image-item"
-                    [src]="
-                      getBaseUrl() +
-                      policy.mainProduct.coverageProduct.images[0]?.url
-                    "
-                    alt=""
-                  />
+<ng-container *ngIf="policies$ | async as policies; else noPolicies">
+  <ng-container *ngIf="policiesLoaded$ | async; else loading">
```

Comment:
First, we should check if the policies are loaded from backend. And then if policies do not exist (angular store is empty) display noPolicies template